services/knative/0.18.3/knative-serving.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: knative
+  namespace: ${releaseNamespace}
+spec:
+  dependsOn:
+    - name: kube-prometheus-stack
```

Comment:
@alembiewski do we still need to remove this?

Reply:
Yes, I think we should remove this to avoid any confusion